File: 151003/Mytnik/Lab2/Lab2/Lab2/DTO/NoteRequestTo.cs
using Lab2.DTO.Interface;

namespace Lab2.DTO
{
    public class NoteRequestTo(int Id,int? TweetId, string? Content) : IRequestTo
    {
        public int Id { get; set; } = Id;
        public int? TweetId { get; set; } = TweetId;

        public string? Content { get; set; } = Content;
    }
}

File: 151003/Alamov/Labs-3-4/discussion/src/main/java/org/example/discussion/impl/note/dto/NoteRequestTo.java
package org.example.discussion.impl.note.dto;

import jakarta.validation.constraints.Size;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.math.BigInteger;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class NoteRequestTo {
    private BigInteger id;
    private BigInteger tweetId;
    @Size(min = 2, max = 2048)
    String content;
}
